x11-packages/xfce4-cpugraph-plugin/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://docs.xfce.org/panel-plugins/xfce4-cpugraph-plugin/start
+TERMUX_PKG_DESCRIPTION="Graphical representation of the CPU load"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Since you wrote a pretty sizable patch for this package, please consider setting yourself as the maintainer.